src/algebra/archimedean.lean

```diff
@@ -370,6 +370,25 @@ begin
     { simp [zero_lt_one] }}
 end
 
+lemma exists_nat_one_div_lt_and [linear_ordered_field α] [archimedean α]
+  {ε₁ ε₂ : α} (hε₁ : ε₁ > 0) (hε₂ : ε₂ > 0) :
+  ∃ n : ℕ, 1 / (n + 1: α) < ε₁ ∧ 1 / (n + 1: α) < ε₂ :=
+let ⟨n₁, h₁⟩ := exists_nat_one_div_lt hε₁ in
```

Comment:
I think it would be easier to prove this by applying `exists_nat_one_div_lt` to `min ε₁ ε₂`. Maybe even so much easier you find it's not really necessary to prove this lemma at all.
If you do want to keep the lemma, then tactic mode can reduce duplication in the proof (using things like `;` and `<|>` where necessary, or `all_goals`).

Reply:
Thank you. It is so much easier.